.jenkins/jenkins_buildbot_mac.sh

```diff
@@ -17,6 +17,9 @@ export PATH=/usr/local/cuda/bin:${PATH}
 export DYLD_LIBRARY_PATH=/usr/local/cuda/lib:${DYLD_LIBRARY_PATH}
 export CPLUS_INCLUDE_PATH=/usr/local/cuda/include:${HOME}/cuda/include:${CPLUS_INCLUDE_PATH}
 
+# CUDNN
+export CUDNNPATH=${HOME}/cuda
```

Comment:
By default, when unpacking the cudnn archive, it generates a root directory called cuda